.github/workflows/docker_build_push.sh

```diff
@@ -47,13 +47,13 @@ if [ -z "${DOCKERHUB_TOKEN}" ]; then
   # Skip if secrets aren't populated -- they're only visible for actions running in the repo (not on forks)
   echo "Skipping Docker push"
   # By default load it back
-  DOCKER_ARGS="--load"
+  DOCKER_ARGS="--load --cache-from=type=registry,ref=apache/superset:cache"
   ARCHITECTURE_FOR_BUILD="linux/amd64 linux/arm64"
 else
   # Login and push
   docker logout
   docker login --username "${DOCKERHUB_USER}" --password "${DOCKERHUB_TOKEN}"
-  DOCKER_ARGS="--push"
+  DOCKER_ARGS="--push --cache-from=type=registry,ref=apache/superset:cache --cache-to type=registry,ref=apache/superset:cache,mode=max"
```

Comment:
I am just curious. Doesn't this always overwrite the same image cache during a single build? E.g. `superset:websocket` overwrites `superset:py310`. And `superset:dockerize` overwrtites `superset:websocket` etc?

Reply:
@sebastianliebscher Thanks for your comment, you were completely right. I did what you said and got the cache overwritten by the new cache.
I eliminated this oversight and now everything should work fine in my opinion.